File: Go/File/write-to-file.go
package main

import (
	"log"
	"os"
)

func main() {
	// Write to file without appending anything.
	writeToFile("foo.txt", "Hello World")
}

// Don't append and write to file
func writeToFile(filepath string, content string) {
	err := os.WriteFile(content, []byte(filepath), 0644)
	if err != nil {
		log.Fatalln(err)
	}
}

File: TypeScript/File/write-to-file.ts
import * as fs from "fs"

function main(): void {
    // Write a string to a given file.
    writeToFile("assets/remove/4G8i6e782HNrDUdh65WS4bp2eKCk5D5i", "Hello, World!")
}

main()

// Write a string to a given file.
function writeToFile(filePath: string, content: string): void {
    fs.writeFileSync(filePath, content)
}